### PR TITLE
Add systemd service file and description

### PR DESCRIPTION
**Pull request by aresius**
I added the systemd service file that I wrote and explain how I am using it on my RaspberryPi with FHEM.

I do not use docker since it is unnecessary and would need another deamon to run on the very limited hardware.

Include it into the package as you like, and thanks a lot for the nice work.

### CODE REVIEW

File: systemd/README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Systemd service file
+
+1. Create or define a user that should run the service
```
Comment (kbialek) [inline systemd/README.md:3]: `useradd` command can be given as an example here

File: systemd/README.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# Systemd service file
```
Comment (kbialek) [inline systemd/README.md:1]: `git clone` is zip download is the very first step.

**Review by kbialek — COMMENTED** [review]
@aresius Thanks for this contribution. You are the 2nd user requesting this, so seems like this is something more desired than I initially thought. Ideally it would be to have this wrapped in an installation script. Would you be able to implement it?

**Comment by aresius** [comment]
> Ideally it would be to have this wrapped in an installation script. Would you be able to implement it?

Yes, I will write one, but not immediately, too busy the next two weeks. I plan to work on it around Easter.

**Comment by kbialek** [comment]
Sounds good!

**Comment by aresius** [comment]
Unfortunately I did not find the time to work on this topic, again. May be take another week or two. Just to let you know that I did not forget about it.

**Comment by aresius** [comment]
Later than expected but not forgotten. I finally managed to take the time to write an install script and here it is.

**Review by kbialek — APPROVED** [review]
LGTM. When someone will report an issue about this I will forward him to you :slightly_smiling_face:

**Pull request merged**